src/commands.js

```diff
@@ -283,9 +286,9 @@ async function sendMessagesForEvents(api, cacheKey, options, action) {
 
     debug(`startDate=${startDate}, endDate=${endDate}`);
     const buckets = helpers.getMonthDatesBetween(startDate, endDate).map(getBucketFromTime);
-    const eventsInBuckets = await helpers.mapPromise(buckets, bucket =>
-        api.get(`/dataStore/${namespace}/${bucket}`).catch(err => [])
-    );
+    const eventsRepository = new EventsRepository();
+    const eventsInBuckets = buckets.map(bucket => eventsRepository.get(bucket));
+
     const triggerEvents = _(eventsInBuckets)
```

Comment:
I am having problems with this, events created in the last hour are filtered out here. I think the problem comes with the format of created in `cache/events/ev-month....json`. Example, it's 13:15 local time and I create an interpretation, the field value is "2020-04-06T13:15:24.100Z", but Z mean UTC, so it's not the correct time. Am I missing something?

Reply:
that's weird, for me It's working. For all changes, email is sent.
if I realized a change at 7:22 local time then the created value in event file is "2020-04-07T05:22:02.778Z"

Reply:
Time-zone can be a bit tricky, may depend on specific instance/OS configuration. For example, in my case, if I create an interpretation at 9:51, that date in DB is: "2020-04-07T09:51:58.573". Ok, it's the local date. This passes to dhisDateToISODate, where gets the trailing "Z", thus the (correct) local time is not a (wrong) UTC time. In your instance it may be different, we'll have to find a way that works everywhere.

Reply:
I checked and both psi.new and play.dhis2 return UTC for timestamps. So my db having localtime is the weird one, I think we can ignore this then.

Reply:
ok